Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml

```diff
@@ -185,14 +150,11 @@
 - type: entity
   parent: [filingCabinet, BaseBureaucraticStorageFill]
```

Comment:
![grafik](https://github.com/user-attachments/assets/6e7642ec-179c-4573-a981-60ab52ff3a8e)
They all show up as Empty if you inherit it in this order

Reply:
hm... BaseBureaucraticStorageFill inherited as the second entity, so it should overwrite the first, idk, ill check